app/views/admin/loans/questionnaires/breakeven_rampup/_rampup_table.html.slim

```diff
@@ -0,0 +1,26 @@
+tr.group-content
+  td
+    td.text-center colspan=periods
+      = units
+      br
+      = "#{t("loan.breakeven_rampup.quantity")} / #{t("loan.breakeven_rampup.price")}"
+
+tr.group-columns
+  td.group-title = title
+  - (1..periods).each do |period|
```

Comment:
I think you need a range for this. If I change the code, I get: `undefined method 'each' for 4:Fixnum`